### PR TITLE
feat: add SlotUsageCheque

### LINKED ISSUE #390: Create the `SlotUsageCheque` table
We need to create a table to track the intent of the usage of a slot. Each record will be created on publish and the table should have:

```ts
id: uuid
signature: string
quantity: string
third_party_id: string
salt: string
collection_id: string
created_at: Date
updated_at: Date
```

When tackling this issue, re-check with the team ( @LautaroPetaccio , @nicosantangelo , @juanmahidalgo ) to see if any requirement changed and new table columns will be needed or old ones removed

Blocking: https://github.com/decentraland/builder-server/issues/389

### PR DESCRIPTION
**Pull request by nicosantangelo**
Closes #390

### CODE REVIEW

File: migrations/1644469112765_create-slot-usage-cheques.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const tableName = SlotUsageCheque.tableName
+
+export const up = (pgm: MigrationBuilder) => {
+  pgm.createTable(
+    tableName,
+    {
+      id: { type: 'UUID', primaryKey: true, unique: true, notNull: true },
+      signature: { type: 'TEXT', notNull: true },
+      quantity: { type: 'TEXT', notNull: true },
+      salt: { type: 'TEXT', notNull: true },
```
Comment (LautaroPetaccio) [inline migrations/1644469112765_create-slot-usage-cheques.ts:13]: Do we need the salt to be a column? Ins't it going to be part of the signed message?

File: migrations/1644469112765_create-slot-usage-cheques.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { SlotUsageCheque } from '../src/SlotUsageCheque'
+
+const tableName = SlotUsageCheque.tableName
+
+export const up = (pgm: MigrationBuilder) => {
+  pgm.createTable(
+    tableName,
+    {
+      id: { type: 'UUID', primaryKey: true, unique: true, notNull: true },
+      signature: { type: 'TEXT', notNull: true },
```
Comment (LautaroPetaccio) [inline migrations/1644469112765_create-slot-usage-cheques.ts:11]: Is it signature or signed message?

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

File: migrations/1644469112765_create-slot-usage-cheques.ts (hunk 2 above)
Comment (nicosantangelo) [reply migrations/1644469112765_create-slot-usage-cheques.ts:11]: signed message sounds better I think

File: migrations/1644469112765_create-slot-usage-cheques.ts (hunk 1 above)
Comment (nicosantangelo) [reply migrations/1644469112765_create-slot-usage-cheques.ts:13]: It is, but we decided on adding it here too when creating https://github.com/decentraland/builder-server/issues/390 with @nachomazzara . I can't really recall if it's necessary now

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**